src/main/java/com/coreos/jetcd/LeaseImpl.java

```diff
@@ -62,158 +65,192 @@
    */
   private StreamObserver<LeaseKeepAliveResponse> keepAliveResponseStreamObserver;
 
+  private boolean firstKeepAlive = true;
```

Comment:
firstKeepAliveSeen = false? this boolean member variable is not super descriptive.